src/notes/patch v3.6.md

```diff
@@ -88,34 +88,34 @@
 - When all items are removed from a crate, the cardboard box that replaces it will match the size of the original crate
 - Harley Filben will still talk to the player on Liberty Island if using Confix (eg. GMDX) and you didn't get the password from Paul first.'
   - Note: Confix is NOT required or recommended for use with vanilla Randomizer
-- Memory Containment Units will be ignored for the purposes of the Unaugmented Prison Pocket setting, allowing you to keep another item.  Memory Containment Units will always be allowed to be kept.
+- Memory Containment Units will be ignored when removing intentory items in M05, even with the Unaugmented Prison Pocket setting
```

Comment:
Your text is technically correct, however your text doesn't actually say what's different in this release from the previous release. You always got to keep your MCUs, the new thing is that Unaugmented Prison Pocket now skips over the MCUs to "stash away" the next item instead.

```suggestion
- Memory Containment Units will be ignored for the purposes of the Unaugmented Prison Pocket setting, allowing you to keep another item.  Memory Containment Units will always be allowed to be kept.
```